.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
 /htdocs/vendor/
+composer.lock
```

Comment:
If we start ignoring composer.lock, should we delete it?

Reply:
Good remark. Thanks for pointing at this issue. I have opened a dedicated issue for discussing about it: #929